actioncable/lib/action_cable/connection/subscriptions.rb

```diff
@@ -22,7 +22,7 @@ def execute_command(data)
         end
       rescue Exception => e
         @connection.rescue_with_handler(e)
-        logger.error "Could not execute command from (#{data.inspect}) [#{e.class} - #{e.message}]: #{e.backtrace.first(5).join(" | ")}"
+        logger.error "Could not execute command from (#{data.inspect}) [#{e.class} - #{e.message.truncate(300)}]: #{e.backtrace.first(5).join(" | ")}"
```

Comment:
I wonder if we should migrate this to `Rails.error.handle` instead 🤔

Reply:
Yes 👍

Reply:
I wonder has Rails already handled internal errors by `Rails.error.handle`? I didn't find any examples in the codebase by a global search 😭

Reply:
See: https://github.com/rails/rails/blob/3b4e476221ab8d385982d0f9cbe9d9c0f76b8166/activesupport/lib/active_support/cache/redis_cache_store.rb#L63-L67

Reply:
Just added the `ActiveSupport.error_reporter&.report` call.

Reply:
I think we should remove the `logger.error` invocation and just use the new standardized interface instead.

This is a good opportunity to replace all the `logger.error` usages in ActionCable and replace them with the new error reporting system. Would you want to tackle that @piecehealth? I can take a stab if not.

Reply:
Hi @jorgemanrubia , I need to take more time to digest the usage of the new error report interfaces, I guess it shouldn't just replace all the `logger.error` with `ActiveSupport.error_reporter&.report`.

Feel free to do the migration PR if you have time, just remember to tag me 😉